<launch>
	  <arg name="mvibot_seri" default=""/>
	  <node name="rplidarNode1"  pkg="rplidar_ros"  type="rplidarNode" output="screen" respawn="true">	  
		  <param name="serial_port"         type="string" value="/dev/my_USB0"/>
		  <param name="serial_baudrate"     type="int"    value="115200"/>
		  <param name="frame_id"            type="string" value="$(arg mvibot_seri)/base_lidar_1"/>
		  <param name="inverted"            type="bool"   value="false"/>
		  <param name="angle_compensate"    type="bool"   value="true"/>
		  <remap from="scan" to="/$(arg mvibot_seri)/laser/scan1"/>
		  
		<!-- <param name="scan_mode"     type="string"  value="Boost"/> -->
	  </node>
</launch>
